packages/odyssey-react/src/components/Table/TableSortButton.tsx

```diff
@@ -23,15 +27,46 @@ import styles from "./Table.module.scss";
 
 export type TableSortDirections = "asc" | "desc" | "unsorted";
 
-export type Props = {
+interface Props
+  extends Omit<ComponentPropsWithoutRef<"button">, "style" | "className"> {
   children?: ReactNode;
+  /**
+   * Current sort direction shown in icon
+   * @default unsorted
+   */
   direction: TableSortDirections;
-} & ComponentProps<"button">;
+  /**
+   * Title for the unsorted icon
+   * @default Unsorted
+   */
+  unsortedIconTitle?: string;
+  /**
+   * Title for the ascending sort icon
+   * @default Ascending
+   */
+  ascendingIconTitle?: string;
+  /**
+   * Title for the descending sort icon
+   * @default Descending
+   */
+  descendingIconTitle?: string;
+  /**
+   * Screen read only text used as the call to action for the button
+   * @default click to sort
+   */
+  screenReaderCallToAction?: string;
+}
 
-type Ref = HTMLButtonElement;
-
-const TableSortButton = forwardRef<Ref, Props>((props, ref) => {
-  const { children, direction = "unsorted", ...rest } = props;
+let TableSortButton = forwardRef<HTMLButtonElement, Props>((props, ref) => {
+  const {
+    children,
+    direction = "unsorted",
+    unsortedIconTitle = "Unsorted",
+    ascendingIconTitle = "Ascending",
+    descendingIconTitle = "Descending",
+    screenReaderCallToAction = "click to sort",
```

Comment:
we shouldn't assign english string defaults here - no user facing english strings anywhere within Odyssey!

Reply:
Reminder that while English defaults presents an i18n _liability_, removing defaults without an error or fallback creates a WCAG _violation_. Gov customers and those with a fed-compliant VPAT are legally obligated to meet this requirement - even in third-party software like Okta. We should not trade one regression for another.

Associated props when the strings are removed should become required until we have an alternate solution for these cases.

Reply:
@edburyenegren-okta thanks! take a look at the types here and you'll see we aren't trading regressions. We're fixing an i18n bug and enforcing that customers provide their own strings. If someone isn't using typescript we can also enforce this at the react prop types layer once we get to that level of quality. tl;dr these should be required today

@andrewberg-okta looks like we still have a hardcoded english string as a default here `"click to sort"`. Lets remove it as we have the prop typed to be required above already.

Reply:
@jeffbelser-okta these types were optional when I left my comment, hence the a11y risk. 

Thank you for updating @andrewberg-okta!